backend/main.py

```diff
@@ -59,7 +59,6 @@ def fastapi_app():
         os.makedirs(path)
 
 
-@app.on_event('startup')
-@repeat_at(cron="* * * * *")
-def start_job():
+@modal_app.function(image=image,schedule=Period(minutes=15))
```

Comment:
Why are we doing it every 15 minutes, would this trigger at 8pm?

Reply:
You said, I should change it to every hour instead of every minute but because of the time zone difference, it is more efficient to run it every 15 minutes.


Some timezones have offset of 15, 30, 45 so running every 15 minutes should handle the case

[Ref](https://help.skytap.com/kb-time-zones-and-utc-offsets.html)

Reply:
Right but if I container starts at 08:05, would it trigger at 08:20? no?

I think we need to do smth like:
`@app.function(schedule=modal.Cron("0 8 * * 1"))`